setup.py

```diff
@@ -354,12 +354,15 @@ def get_path(*filepath) -> str:
 
 
 def get_vllm_version() -> str:
-    version = get_version()
+    version = get_version(
+        write_to="vllm/_version.py", # TODO: move this to pyproject.toml
+    )
+
     sep = "+" if "+" not in version else "."  # dev versions might contain +
 
     if _no_device():
         if envs.VLLM_TARGET_DEVICE == "empty":
-            version += "+empty"
+            version += f"{sep}empty"
```

Comment:
Not properly setting this breaks builds because of a duplicated `+`, in the comment on line 361